tiny_dnn/core/kernels/conv2d_op_internal.h

```diff
@@ -72,41 +80,51 @@ inline void conv2d_op_internal(const tensor_t &in_data,
 }
 
 /******************************************************************/
+template <typename S1,
+          typename S2,
+          typename S3,
+          typename S4,
+          typename S5,
+          typename S6>
```

Comment:
@Randl It was just a vague idea. After the integration will be finished, let's consider how we can improve the code.